dsgrid/cli/registry.py

```diff
@@ -988,6 +998,149 @@ def update_dataset(
         return 1
 
 
+@click.command()
+@click.argument("registration_file", type=click.Path(exists=True))
+@click.option(
+    "-d",
+    "--base-data-dir",
+    type=click.Path(exists=True),
+    callback=_path_callback,
+    help="Base directory for input data. If set, and if the dataset paths are relative, prepend "
+    "them with this path.",
+)
+@click.option(
+    "-r",
+    "--base-repo-dir",
+    type=click.Path(exists=True),
+    callback=_path_callback,
+    help="Base directory for dsgrid project/dataset repository. If set, and if the config file "
+    "paths are relative, prepend them with this path.",
```

Comment:
This is making an assumption about the structure of repositories like https://github.com/dsgrid/dsgrid-project-DECARB?

Reply:
I see ... not really since this is just being applied to the file paths listed in the bulk registration .json.

Reply:
Correct. It allows for a git-committed-file to work for different users.